### PR TITLE
net: phy: broadcom: Make LEDs 3+4 shadow LEDs 1+2 [REVPI-3119]

### PR DESCRIPTION
**Pull request by nbuchwitz**
commit 326dea65f88e75d25960ecce49df23a0d76bbac0 upstream (rpi)

CM4 uses BCM54210PE, which supports 2 additional LEDs, choosing LED3 for the amber LED because it shows activity by default (LED4 is not connected). However, this makes it uncontrollable by the eth_led<n> dtparams which target LEDs 1+2.

Solve the problem by making LEDs 3+4 mirror LEDs 1+2 (which is much simpler than adding baseboard-specific overrides, but comes with a risk of making one of the LEDs redundant).

See: https://github.com/raspberrypi/linux/issues/5289

### CODE REVIEW

**Comment by nbuchwitz** [comment]
This backported patch from `rpi-5.15.y` in combination with

```
dtparam=eth_led0=8
dtparam=eth_led1=0
```

in `/boot/config.txt` will make the CM4 ethernet LEDs behave like all other RevPi ethernet LEDS (amber=activity, green=link).

**Comment by l1k** [comment]
I'm a little confused that you're both author and committer. Normally if you use `git cherry-pick`, the author of the upstream commit is retained and you just become the committer. FWIW it's possible to change this after the fact with `git commit --amend --author=...`, then force-push. You can verify the author and committer is the way you want it with `git log --pretty=fuller`.

**Comment by nbuchwitz** [comment]
@l1k Right, that looks odd. Must be something gone wrong during the cherry-pick and fix afterwards. Anyway, I've did a quick reset, cherry-pick again and now it looks as it (hopefully) should look like.

**Review by l1k — APPROVED** [review]
I have no objections but you may want to wait for comments from @iluminat23 and others before merging.

**Review by iluminat23 — APPROVED** [review]
I really dislike these ugly hacks. But as it is in our next kernel anyway. We can adopt this fuckup now

**Pull request merged**
